### PR TITLE
[null-safety] remove mockito from scrollbar test

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

More work towards https://github.com/flutter/flutter/issues/62886

### CODE REVIEW

File: packages/flutter/test/widgets/scrollbar_test.dart [hunk 1]
```diff
@@ -39,12 +40,20 @@ ScrollbarPainter _buildPainter({
...
+    rects.add(rect);
+  }
+}
 
 void main() {
   final _DrawRectOnceCanvas testCanvas = _DrawRectOnceCanvas();
   ScrollbarPainter painter;
-  Rect captureRect() => verify(testCanvas.drawRect(captureAny, any)).captured.single as Rect;
+
+  Rect captureRect() => testCanvas.rects.removeLast();
```
Comment (goderbauer) [inline packages/flutter/test/widgets/scrollbar_test.dart:56]: `.single` instead of `removeLast()` to capture that there's only been one call?

Comment (jonahwilliams) [reply packages/flutter/test/widgets/scrollbar_test.dart:56]: I don't think thats how verify works, the verification result is created once.

Comment (jonahwilliams) [reply packages/flutter/test/widgets/scrollbar_test.dart:56]: I guess we could clear the state between each test run

File: packages/flutter/test/widgets/scrollbar_test.dart [hunk 2]
```diff
@@ -53,9 +53,12 @@ void main() {
   final _DrawRectOnceCanvas testCanvas = _DrawRectOnceCanvas();
   ScrollbarPainter painter;
 
-  Rect captureRect() => testCanvas.rects.removeLast();
+  Rect captureRect() => testCanvas.rects.single();
```
Comment (goderbauer) [inline packages/flutter/test/widgets/scrollbar_test.dart:56]: I believe single is a property and not a method.

Comment (jonahwilliams) [reply packages/flutter/test/widgets/scrollbar_test.dart:56]: darn my attempts to edit via github UI

**Review by goderbauer — APPROVED** [review]
LGTM, but checks are not passing :(

**Comment by jonahwilliams** [comment]
yeah, I think that is not how verify works. Its a single, but only for the matching invocation - of which there is only one

**Pull request merged**